config.go

```diff
@@ -1908,7 +1908,7 @@ func extractBitcoindRPCParams(networkName string,
 		dataDir = string(dataDirSubmatches[1])
 	}
 
-	chainDir := ""
+	var chainDir string
```

Comment:
Ha interesting.

Reply:
Yeah, quite interesting. The linter now complains about "the initial value not being used".